.github/pipeline-descriptor.yml

```diff
@@ -0,0 +1,30 @@
+github:
+  username: ${{ secrets.JAVA_GITHUB_USERNAME }}
+  token:    ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+codeowners:
+- path:  "*"
+  owner: "@paketo-buildpacks/java-maintainers"
+
+package:
+  repositories:   ["docker.io/paketobuildpacks/pipeline-builder-stool-pigeon","gcr.io/paketo-buildpacks/pipeline-builder-stool-pigeon"]
+  register:       true
+  registry_token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+docker_credentials:
+  - registry: gcr.io
+    username: _json_key
+    password: ${{ secrets.GCR_PUSH_BOT_JSON_KEY }}
+  - registry: docker.io
+    username: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_USERNAME }}
+    password: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD }}
+
+dependencies:
+- id:   maven
+  uses: docker://ghcr.io/paketo-buildpacks/actions/maven-dependency:main
+  with:
+    uri:         https://repo1.maven.org/maven2
+    group_id:    org.apache.maven
+    artifact_id: apache-maven
+    classifier:  bin
+    packaging:   tar.gz
```

Comment:
I think this can probably go away. This was just so that we could test dependency update jobs. I think that's covered with pipeline-builder-canary, and also I don't think that's really a concern of extensions.

After removing this, we should be able to delete the `pb-update-maven.yml` file as well.

Reply:
I figured that extensions can have dependencies just like buildpacks (say archives that are being applied with root level perms to system folders).. and maven was as good a choice as any to just make sure I hadn't broken the support for the updates etc...

Reply:
OK, if you think that's a case worth covering I'm 👍